src/components/application_manager/include/application_manager/commands/mobile/send_location_request.h

```diff
@@ -84,6 +84,18 @@ class SendLocationRequest : public CommandRequestImpl {
    */
   bool IsWhiteSpaceExist();
 
+  /**
+   * @brief Check policy permissions of mandatory parameters
+   * @return false in case when one of mandatory params disallowed by policy
+   */
+  bool AreMandatoryParamsAllowedByPolicy() const;
+
+  /**
+   * @brief Check policy permissions of mandatory parameters
+   * @return false in case when one of mandatory params disallowed by user
```

Comment:
@LitvinenkoIra return false if parameters (all parameters, some of parameters ? ) are not listed as allowed), otherwise return true